.travis.yml

```diff
@@ -16,6 +16,7 @@ before_install:
   - sudo apt-get install -y build-essential libcap-dev
 install:
   - pip install tox
+  - pip install -r test_requirements.txt
```

Comment:
Agree. Also I would request either renaming `test_requirements.txt` to `requirements_test.txt` in sideboard, or renaming `requirements_test.txt` to `test_requirements.txt` in all of our other plugins.